tur/luanti/build.sh

```diff
@@ -2,31 +2,13 @@ TERMUX_PKG_HOMEPAGE=https://www.luanti.org
 TERMUX_PKG_DESCRIPTION="An open source voxel game engine."
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux-user-repository"
-TERMUX_PKG_VERSION=2024.11.03
-_COMMIT=9982c563730f294d404119a2b8ffe26073134884
-TERMUX_PKG_SRCURL=git+https://github.com/minetest/minetest.git
-TERMUX_PKG_GIT_BRANCH=master
-TERMUX_PKG_SHA256=0aa7f0b08de6e7b74c522f62a354e88f8a291da53c4b0e38ca4f4cb10f6d2e86
+TERMUX_PKG_VERSION=5.10.0
```

Comment:
Should bump epoch as `2024.11.03` > `5.10.0`
```suggestion
TERMUX_PKG_VERSION=1:5.10.0
```

Reply:
Thanks for the tip, I applied that change